### ISSUE TITLE
update intl

### ISSUE THREAD

**Issue by ivanzhadnov** [bluefireteam/daylight#12]
Because daylight 2.0.0 depends on intl ^0.17.0 and no versions of daylight match >2.0.0 <3.0.0, daylight ^2.0.0 requires intl ^0.17.0.
So, because XXX depends on both intl ^0.18.0 and daylight ^2.0.0, version solving failed.

**Comment by RaphaelArabeyre** [comment]
import package using git : 
``` YAML
dependencies:
  ...
  daylight:
    git:
      url: https://github.com/bluefireteam/daylight.git
  ...
````

**Comment by renancaraujo** [comment]
Fixed on v3.0.0
